<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_view</class>
 <widget class="QWidget" name="main_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>20</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_menu">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>460</y>
     <width>96</width>
     <height>96</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Панель навигации"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/res/icons/menu.svg</normaloff>:/res/icons/menu.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
  </widget>
  <widget class="QcGaugeWidget" name="gau_spd" native="true">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>70</y>
     <width>380</width>
     <height>380</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>380</height>
    </size>
   </property>
  </widget>
  <widget class="QcGaugeWidget" name="gau_wgt" native="true">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>140</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>280</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>280</height>
    </size>
   </property>
  </widget>
  <widget class="QcGaugeWidget" name="gau_flv" native="true">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>140</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>280</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>280</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СМЕННОЕ ЗАДАНИЕ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>10</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ДВИЖЕНИЕ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>271</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ТОПЛИВО</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>271</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>МАССА</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcGaugeWidget</class>
   <extends>QWidget</extends>
   <header>Utils/Widgets/QcGauge/QcGaugeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
